package.json

```diff
@@ -94,6 +96,7 @@
     "pino": "6.6.1",
     "pino-elasticsearch": "5.3.0",
     "pino-pretty": "4.2.1",
+    "react-datalist-field": "20.3.1",
```

Comment:
Go to `src/frontend` and run `npm install --save react-datalist-field` and remove this from the root `package.json` (these are only for the backend).

Reply:
Done.